sirepo/srunit.py

```diff
@@ -153,6 +153,24 @@ def __init__(self, env, job_run_mode, port):
         if feature_config.cfg().ui_websocket:
             self._websocket = _WebSocket(self)
 
+    def assert_post_will_redirect(self, expect_re, *args, **kwargs):
```

Comment:
The goal is to avoid duplication of the `args`. Really this should be a dispatch. I want to change it to assert_op_will_redirect with the arguments being: `expect_re, method, *args, **kwargs`. This method is simply a wrapper so it shouldn't define the args.